wsproto/frame_protocol.py

```diff
@@ -32,7 +32,25 @@ def process(self, data):
 
 
 # RFC6455, Section 5.2 - Base Framing Protocol
-MAX_FRAME_PAYLOAD = 2 ** 64
+
+# Payload length constants
+PAYLOAD_LENGTH_TWO_BYTE = 126
+PAYLOAD_LENGTH_EIGHT_BYTE = 127
+MAX_PAYLOAD_NORMAL = 125
+MAX_PAYLOAD_TWO_BYTE = 2 ** 16
+MAX_PAYLOAD_EIGHT_BYTE = 2 ** 64
+MAX_FRAME_PAYLOAD = MAX_PAYLOAD_EIGHT_BYTE
```

Comment:
MAX_FRAME_PAYLOAD existed previously but I can't see it used anywhere. I can remove them or they can stay there in case they become relevant.